src/MAT.jl

```diff
@@ -85,9 +85,10 @@ function matopen(filename::AbstractString, rd::Bool, wr::Bool, cr::Bool, tr::Boo
 end
 
 function matopen(fname::AbstractString, mode::AbstractString)
-    mode == "r"  ? matopen(fname, true , false, false, false, false) :
-    mode == "r+" ? matopen(fname, true , true , false, false, false) :
-    mode == "w"  ? matopen(fname, false, true , true , true , false) :
+    mode == "r"  ? matopen(fname, true , false, false, false, false, false) :
+    mode == "r+" ? matopen(fname, true , true , false, false, false, false) :
+    mode == "w"  ? matopen(fname, false, true , true , true , false, false) :
+    mode == "wz" ? matopen(fname, false, true , true , true , false, true ) :
```

Comment:
Is this standard? Should be documented in any case.

Seems like `z` might be relevant for `r+`? If so, you could check `compress = endswith(mode, 'z')` and if true strip it off and then pass `compress` as the third argument.

Reply:
> Is this standard?
I took this from fguevaravas original PR tbh.

Maybe a keyword argument like `matopen(...; compress = true)` both for matopen and matwrite would be best?

Would you prefer keyword argument or "z" options for the mode-string?

Reply:
If the `wz` isn't standard, then I think the keyword might be best.

Reply:
On it!